### PR TITLE
feat(FileUploaderItem): Replaces Icon component with transparent icon Button component.

### PR DESCRIPTION
**Pull request by adityajhajharia**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/molecules/fileUploader/FileUploaderStatus.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { ProgressRing, Icon } from '@/index';
 import { FileStatus } from './FileUploaderItem';
-
+import Button from '@/components/atoms/button/Button';
```
Comment (satyamyadav) [inline core/components/molecules/fileUploader/FileUploaderStatus.tsx]: Please update import statement to `import { Button } from '@/index'`;

File: core/components/molecules/fileUploader/FileUploaderStatus.tsx [hunk 2]
```diff
@@ -25,11 +25,13 @@ export const FileUploaderStatus = (props: FileUploaderStatusProps) => {
 
     case 'error':
       return (
-        <Icon
+        <Button
+          appearance="transparent"
+          size="regular"
           name="refresh"
```
Comment (satyamyadav) [inline core/components/molecules/fileUploader/FileUploaderStatus.tsx]: Please remove `name` prop.

File: core/components/molecules/fileUploader/FileUploaderStatus.tsx [hunk 3]
```diff
@@ -25,11 +25,13 @@ export const FileUploaderStatus = (props: FileUploaderStatusProps) => {
...
-          size={20}
           onClick={onRetry}
           className="mr-4 cursor-pointer"
```
Comment (satyamyadav) [inline core/components/molecules/fileUploader/FileUploaderStatus.tsx]: Please remove `cursor-pointer` class.

File: core/components/molecules/fileUploader/FileUploaderItem.tsx [hunk 4]
```diff
@@ -77,11 +78,13 @@ export const FileUploaderItem = (props: FileUploaderItemProps) => {
...
           />
-          <Icon
+          <Button
             name="close"
-            size={20}
+            appearance="transparent"
+            size="regular"
             onClick={() => onDelete && onDelete(file, id)}
-            className="py-2 px-2 my-3 mx-3 cursor-pointer"
+            className="py-2 px-2 my-3 mx-3 cursor-pointer "
```
Comment (satyamyadav) [inline core/components/molecules/fileUploader/FileUploaderItem.tsx]: Please remove `cursor-pointer` class. Also avoid using padding classes on button.

**Review by riyalohia — DISMISSED** [review]

**Review by satyamyadav — CHANGES_REQUESTED** [review]
Please update title and commit of the pull request to `feat(FileUploaderItem): Replaces Icon component with transparent icon Button component.
`

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**